### PR TITLE
Fix cors policy problems in bluespice

### PR DESCRIPTION
**Pull request by nfranzeck**
- avoid inline script execution by using src in script tag
- add versions to fetch add-warp-menu.js and add-whitelabeling.js

### CODE REVIEW

File: resources/var/www/html/warpmenu/add-warp-menu.js.tpl [hunk 1]
```diff
@@ -7,11 +7,10 @@ const addWarpMenu = function(){
     const urlScript = document.createElement("script");
     // This variable is used inside the warp menu script
     // Update this version whenever warp menu is updated
```
Comment (kahoona77) [inline resources/var/www/html/warpmenu/add-warp-menu.js.tpl:9]:
```suggestion
```
Move down to variable

File: resources/var/www/html/warpmenu/add-warp-menu.js.tpl [hunk 2]
```diff
@@ -7,11 +7,10 @@ const addWarpMenu = function(){
...
-    urlScript.innerHTML = 'const cesWarpMenuWarpCssUrl = "/warp/warp.css?nginx={{ .Env.Get "VERSION"}}";';
 
     const firstScriptTag = document.getElementsByTagName("script")[0];
-    firstScriptTag.parentNode.insertBefore(urlScript, firstScriptTag);
     firstScriptTag.parentNode.insertBefore(warpScript, firstScriptTag);
 }
 
+const cesWarpMenuWarpCssUrl = "/warp/warp.css?nginx={{ .Env.Get "VERSION"}}";
```
Comment (kahoona77) [inline resources/var/www/html/warpmenu/add-warp-menu.js.tpl:15]:
```suggestion
// This variable is used inside the warp menu script
// Update this version whenever warp menu is updated
const cesWarpMenuWarpCssUrl = "/warp/warp.css?nginx={{ .Env.Get "VERSION"}}";
```

File: resources/var/www/html/warpmenu/add-warp-menu.js.tpl [hunk 3]
```diff
@@ -7,11 +7,10 @@ const addWarpMenu = function(){
     const urlScript = document.createElement("script");
```
Comment (kahoona77) [inline resources/var/www/html/warpmenu/add-warp-menu.js.tpl:7]:
```suggestion
```
not needed anymore

**Review by kahoona77 — CHANGES_REQUESTED** [review]
Also add a changelog

**Comment by nfranzeck** [comment]
Ok, I haven committed the suggestions.

File: CHANGELOG.md [hunk 4]
```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Fixed
+- Fix problems with CORS policies caused by whitelabeling
```
Comment (kahoona77) [inline CHANGELOG.md:10]:
```suggestion
- Fix problems with content security policies (CSP) caused by whitelabeling
```

**Review by kahoona77 — APPROVED** [review]
LGTM

**Pull request merged**